app/controllers/blogs_controller.rb

```diff
@@ -3,13 +3,15 @@
 class BlogsController < ApplicationController
   skip_before_action :authenticate_user!, only: %i[index show]
 
-  before_action :set_blog, only: %i[show edit update destroy]
+  before_action :set_current_user_blog, only: %i[edit update destroy]
```

Comment:
```suggestion
  before_action :set_blog, only: %i[edit update destroy]
```

とした方がdiffは小さくなります。（が、どっちがいいかは好みの問題かも）